### DISCUSSION TITLE
Call for translations!

### DISCUSSION THREAD

**Discussion by mozzius** [mozzius/graysky#435] [General]
Graysky was recently translated into Japanese by some generous community contributors (thanks again to them!)

Now that the infrastructure for translating the app is set up, it's now possible to accept translation PRs if you wish to translate the app into your language. If you're interested in doing so, please comment below which language you'd like to tackle, and I'll set up the translation files for you and provide a link to crowdin, where you're able to translate all of the strings via a simple web interface.

Thank you so much to this awesome community, I can't tell you how much it means to me that people are willing to help with this ❤️

**Comment by bielarusajed** [comment]
Hi, I would like to help with Belarusian translation

**Reply by mozzius** [reply]
Awesome!
Here's the link
https://crowdin.com/editor/graysky/8/en-be

Could you also let me know if "беларуская мова" is the correct term to use here?
<img width="388" alt="Screenshot 2024-02-20 at 00 13 30" src="https://github.com/mozzius/graysky/assets/10959775/95584b30-136c-4937-ab2b-c9102e16aec1">

**Reply by bielarusajed** [reply]
Just "Беларуская" would be better

**Comment by ghost** [comment]
Hello, I could help out with Spanish

**Reply by mozzius** [reply]
Awesome! here's the link:
https://crowdin.com/project/graysky/es-ES

**Reply by ghost** [reply]
Great! I'll get working on it

**Comment by YanYannoche** [comment]
Hello ;-)
I can help with french ^_-

**Reply by mozzius** [reply]
Merci!
Here's french:
https://crowdin.com/project/graysky/fr

**Reply by mozzius** [reply]
Wow you're basically done, awesome!

Btw, sorry, make sure you use this link:
https://crowdin.com/editor/graysky/8/en-fr?view=comfortable&filter=basic&value=3
(it shows you two files but you only need to do this one)

**Reply by YanYannoche** [reply]
Okay, sry, I'll bookmark it ;-)

**Reply by mozzius** [reply]
Sorry, it's my fault! I think I set it up wrong

**Reply by YanYannoche** [reply]
There are some untranslated words in the "moderation" tab I can't find in the main file...
![Screenshot_20240220_221848_Graysky](https://github.com/mozzius/graysky/assets/160554736/a00b75aa-ca15-463c-a0ef-c361f6478ec4)

**Reply by YanYannoche** [reply]
Is there a place where we can show the on-screen issues? Or I can post images here?

Anyway, I post it here not to forget, you can delete it if needed ^_-
There is a display issue here : 
![Screenshot_20240220_222547_Graysky](https://github.com/mozzius/graysky/assets/160554736/89eb90b6-3981-4ceb-b11a-183a803a50c6)

**Reply by mozzius** [reply]
1. That is untranslated for now since I want to replace it soon.

2. Noted, thanks. You can open new issues here on GitHub if you spot anything else

**Reply by YanYannoche** [reply]
Okay thank you ;-)

**Comment by garrucho** [comment]
Hi! I can help with Brazilian Portuguese. =)

**Reply by mozzius** [reply]
Awesome, here's the link:
https://crowdin.com/editor/graysky/8/en-ptbr?view=comfortable&filter=basic&value=3
Thanks!

**Reply by garrucho** [reply]
@mozzius Some strings can be weird while not applied in context, so I should review some of them after finished and published. 😅

**Reply by mozzius** [reply]
You can review in this alpha release:
https://github.com/mozzius/graysky/releases/tag/v1.4.6-alpha.2

**Comment by Waldmensch1** [comment]
Hi, I can help with German language. 🙂

**Reply by mozzius** [reply]
Danke!
https://crowdin.com/editor/graysky/8/en-de?view=comfortable&filter=basic&value=3

**Reply by mozzius** [reply]
If you want to review, here's the testflight (android soon) https://testflight.apple.com/join/zZcRtJfy

**Reply by mozzius** [reply]
Android: https://github.com/mozzius/graysky/releases/tag/v1.4.6-alpha.3

**Comment by abhijithgopan** [comment]
Hey, I can help with Malayalam language.

**Reply by mozzius** [reply]
Great, thank you!
https://crowdin.com/editor/graysky/8/en-mlin?view=comfortable&filter=basic&value=3

**Reply by mozzius** [reply]
Amazing work, thank you! Here's the testflight for iOS (will make one for Android shortly)

**Reply by mozzius** [reply]
https://testflight.apple.com/join/zZcRtJfy

**Reply by abhijithgopan** [reply]
Nice, I am using an Android so I will wait.

**Reply by mozzius** [reply]
Android here: https://github.com/mozzius/graysky/releases/tag/v1.4.6-alpha.3

**Reply by abhijithgopan** [reply]
I made some corrections on crowdin after reviewing. 
![Screenshot_20240222-200638~2](https://github.com/mozzius/graysky/assets/46196563/22c5c84c-c9d3-40de-aaa2-e9aa25d9a8bd)
![Screenshot_20240222-200659~2](https://github.com/mozzius/graysky/assets/46196563/ec218a4b-e49e-40f7-a1a0-8cc5a9e0830d)
Also the text for "Posts" is not showing properly. 
The text in the report account function does not give context as it's not showing complete text(this problem is present in the English version too)

**Reply by mozzius** [reply]
1. Fixed
<img width="301" alt="Screenshot 2024-02-22 at 16 31 46" src="https://github.com/mozzius/graysky/assets/10959775/114ba556-1166-4d3d-b4e7-452f16b1772c">

2. yes, I think I need a different solution here

**Comment by comradekingu** [comment]
Happy to help if you use Weblate instead of Crowdin :)
